README.md

```diff
@@ -89,13 +119,15 @@ prediction = model.predict(len(val))
 
 Plot:
 ```python
+import matplotlib.pyplot as plt
+
 series.plot(label='actual', lw=3)
 prediction.plot(label='forecast', lw=3)
 plt.legend()
 plt.xlabel('Year')
 ```
 
-![example](https://github.com/unit8co/darts/raw/develop/static/images/example.png "example") { width=100% }
+<img src="https://github.com/unit8co/darts/raw/develop/static/images/example.png" alt="darts forecast example" width="100%"/>
```

Comment:
Personally, I think 100% might be a bit much

Reply:
It also felt odd on my side in particular the scaling. I centered it instead let me know if its better,

Reply:
Yeah looks much better to me!